packages/rocketchat-lib/server/methods/filterBadWords.js

```diff
@@ -0,0 +1,25 @@
+
+var Filter = Npm.require('bad-words');
+    
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2433000388/issues/source?bid=3199634&fileBranchId=3240346#l3)